group-coordinator/src/main/java/org/apache/kafka/coordinator/group/Group.java

```diff
@@ -166,4 +166,20 @@ void validateOffsetFetch(
      * @return true if the state includes, false otherwise.
      */
     boolean isInStates(Set<String> statesFilter, long committedOffset);
+
+    /**
+     * Returns true if the member exists.
+     *
+     * @param memberId The member id.
+     *
+     * @return A boolean indicating whether the member exists or not.
+     */
+    boolean hasMember(String memberId);
+
+    /**
+     * Returns number of members in the group.
+     *
+     * @return The number of members.
+     */
+    int numMembers();
```

Comment:
nit: `memberCount` clearer maybe? (I instantly think of partitionCount used elsewhere, sounds clearer to me)

Reply:
Yeah may be, but just wanted to have minimal changes in refactoring and re-used existing i.e. ConsumerGroup already had `numMembers()` which I used for Classic Group as well. So I ll leave this like that for now. Incase we need to do that then will open minor refactor PR which has specific changes for renaming it.